File: UI/ebarbershop_admin/lib/models/novost.dart
// TODO Implement this library.
import 'package:json_annotation/json_annotation.dart';

part 'novost.g.dart';

@JsonSerializable()
class Novost {
  int? novostId;
  String? naslov;
  String? sadrzaj;
  DateTime? datumObjave;
  String? slika;
  Novost(
      {this.novostId, this.naslov, this.sadrzaj, this.datumObjave, this.slika});

  factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);
  Map<String, dynamic> toJson() => _$NovostToJson(this);
}

// [
//   {
//     "novostId": 4,
//     "naslov": "string",
//     "sadrzaj": "string",
//     "datumObjave": "2024-12-24T09:53:37.76",
//     "slika": "string"
//   }
// ]

File: eBarbershop.Services/Database/Novost.cs
using System;
using System.Collections.Generic;

namespace eBarbershop.Services.Database;

public  class Novost
{
    public int NovostId { get; set; }

    public string Naslov { get; set; } = null!;

    public string Sadrzaj { get; set; } = null!;

    public DateTime DatumObjave { get; set; }

    public string Slika { get; set; }
}
